### PR TITLE
Fix up skeleton setup + Add Github action for tests

### LINKED ISSUE #2: Add GitHub actions
I have the freaking badge in the readme and it assumes you have tests and Tlint and PHPCS but there's no GitHub actions included by default! C'mon man!

### PR DESCRIPTION
**Pull request by tammytee**
## Summary 

> Fixes #2 and #3 

This PR will fix the illegal byte sequence error that occurs when running the setup script and adds a skeleton github workflow file for running tests

**Successful Github Action Workflow After Setup**

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # :package_fullname
 
 [![Latest Version on Packagist](https://img.shields.io/packagist/v/tightenco/:package_name.svg?style=flat-square)](https://packagist.org/packages/tightenco/:package_name)
-[![GitHub Tests Action Status](https://img.shields.io/github/workflow/status/tighten/:package_name/run-tests?label=tests)](https://github.com/tighten/:package_name/actions?query=workflow%3Arun-tests+branch%3Amain)
+[![GitHub Tests Action Status](https://img.shields.io/github/actions/workflow/status/tighten/:package_name/.github/workflows/tests.yml?branch=main&label=tests)](https://github.com/tighten/:package_name/actions?query=workflow%3Arun-tests+branch%3Amain)
```
Comment (mattstauffer) [inline README.md:4]: Dang, I saw the note from them about changing this but didn't realize we also had to add the `.github/workflows` folder. Good catch!

File: setup.sh [hunk 2]
```diff
@@ -42,29 +42,32 @@ echo    "----------------------------------------------------------------------"
...
 echo
 
-find . -type f -exec sed -i '' -e "s/:author_name/$author_name/g" {} \;
-find . -type f -exec sed -i '' -e "s/:author_username/$author_username/g" {} \;
-find . -type f -exec sed -i '' -e "s/:author_email/$author_email/g" {} \;
-find . -type f -exec sed -i '' -e "s/:package_name/$package_name/g" {} \;
-find . -type f -exec sed -i '' -e "s/:package_fullname/$package_name/g" {} \;
-find . -type f -exec sed -i '' -e "s/:package_description/$package_description/g" {} \;
-find . -type f -exec sed -i '' -e "s/:package_php_namespace/$package_php_namespace/g" {} \;
+LC_ALL=C find . -type f -exec sed -i '' -e "s/:author_name/$author_name/g" {} \;
```
Comment (mattstauffer) [inline setup.sh:55]: Does this have to happen before each line? Did you get a chance to try defining it once in a line above these and seeing if it applies to all?

**Review by mattstauffer — CHANGES_REQUESTED** [review]
Testing it locally now, but here are my notes on just the code :)

File: setup.sh (hunk 2 above)
Comment (tammytee) [reply setup.sh:55]: @mattstauffer - just tried. Looks like it has to be on each line

Comment (mattstauffer) [reply setup.sh:55]: OK, thanks for checking!

**Comment by mattstauffer** [comment]
Fantastic, thank you @tammytee!!

File: skeleton.workflow.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+          coverage: none
+
+      - name: Install dependencies
+        uses: "ramsey/composer-install@v2"
+        with:
+          dependency-versions: "highest"
+          composer-options: "--ignore-platform-reqs --no-plugins"
+
+      - name: Run PHP tests
+        run: vendor/bin/phpunit
```
Comment (driftingly) [inline skeleton.workflow.yml:32]:
```suggestion
        run: vendor/bin/phpunit

```

**Review by driftingly — APPROVED** [review]
Looks good!  

We should consider converting the bash to PHP.  I did a lot of bash development with Duster but I was more productive once I switched over to PHP.

**Pull request merged**